walkthroughs/howto-k8s-cross-cluster/README.md

```diff
@@ -20,7 +20,9 @@ Front app acts as a gateway that makes remote calls to colorapp. Front app has s
 
 *MESH*
```

Comment:
Does this command work? `kubectl -n appmesh-demo run -it curler --image=tutum/curl /bin/bash` - I think we need to explicity disable sidecar injection, else the injector will try to block the creation of a pod without a corresponding virtualnode by default in namespace `howto-k8s-cross-cluster`